dockers/docker-database/start.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+mkdir -p /var/run/redis/
+if [ -f /etc/sonic/database_config.json ]; then
+    cp /etc/sonic/database_config.json /var/run/redis/
+else
+    cp /usr/share/sonic/config/database_config.json /var/run/redis/
```

Comment:
>/var/run/redis [](start = 52, length = 14)

Make sure this directory is readonly. suggest /var/run/sonic-db.

Reply:
modified as required